<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>805</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Travel With Me</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    border-image: url(:/img/BGMAIN.jpg);
}

QFrame#frame_2,
#frame_3 {
    background-color: #f4f5fc;
    border: 1px solid black;
    border-radius: 20px;
}

#L1 {

    background: transparent;
    background-color: #24293e;
    color: white;
    border: transparent;
    border-top-left-radius: 20px;
    border-bottom-left-radius: 20px;
    margin-top: 10px;
    margin-bottom: 10px;
    margin-left: 80px;

}

#R1 {

    background: transparent;
    background-color: #24293e;
    color: white;
    border: transparent;
    border-top-right-radius: 20px;
    border-bottom-right-radius: 20px;
    margin-top: 10px;
    margin-bottom: 10px;
    margin-right: 80px;

}

QPushButton:hover {
    font-weight: bold;
}

#L2 {

    background: transparent;
    background-color: #24293e;
    color: white;
    border: transparent;
    border-top-left-radius: 20px;
    border-bottom-left-radius: 20px;
    margin-top: 10px;
    margin-bottom: 10px;
    margin-left: 80px;

}

#R2 {

    background: transparent;
    background-color: #24293e;
    color: white;
    border: transparent;
    border-top-right-radius: 20px;
    border-bottom-right-radius: 20px;
    margin-top: 10px;
    margin-bottom: 10px;
    margin-right: 80px;

}

QLineEdit {
    background-color: transparent;
    font: 75 italic 10pt &quot;Gill Sans MT&quot;;
    color: black;
    border: none;
    border: 1.75px solid black;
    border-color: #24293e;
    border-radius: 20px;
    margin-top: 10px;
    margin-bottom: 10px;
    margin-right: 80px;
    margin-left: 80px;
    padding-left: 30px;

}

#albtn ,#cabtn{
    background: transparent
}

#registerbtn,#loginb {
    background-color: #FFFFFF;
    border: 1px solid hsl(0, 0%, 13%);
    border-radius: 8px;
    color: #222222;
    margin-left: 200px;
    margin-right: 200px;



}


#TITLE,
#TITLE_2 {
    color: #24293e;
    font: 75 20pt &quot;Georgia&quot;;
}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>29</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>175</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>174</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginpage">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="L1">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="R1">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>10000</width>
                 <height>70</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="TITLE">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>WELCOME</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="username">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="placeholderText">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="password">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="loginb">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cabtn">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>CREATE ACCOUNT</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="registerpage">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="frame_3">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="L2">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Login</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="R2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>70</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>70</height>
                </size>
               </property>
               <property name="sizeIncrement">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Register</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="TITLE_2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>WELCOME</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="QLineEdit" name="register_username">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="placeholderText">
                  <string>Username</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="register_password">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Password</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="confirm_password">
                 <property name="minimumSize">
                  <size>
                   <width>50</width>
                   <height>70</height>
                  </size>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Confirm Password</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="registerbtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>14</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Register</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="albtn">
             <property name="font">
              <font>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>ALREADY HAVE ACCOUNT?</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
